src/styles/settings.css

```diff
@@ -6,7 +6,8 @@ body {
   display: none;
 }
 
-input[type="number"], input[type="time"]{
```

Comment:
formatting and other layout changes should have been in separete PR instead to increase the possibility for them to accept the feature.